abdulfatai360/banka
# #164976342 Add User Dashboard Template

#### What does this PR do?
Build out the UI template for the user dashboard

#### Description of Task to be completed?
Create corresponding and necessary `.html`, `.css`, and `.js` files

#### How should this be manually tested?
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto our local machine
- Navigate to the repo by running `cd banka/` on terminal
- Ensure your working directory is clean by running  `git status`  command on terminal
- Get a copy of the PR by typing  `git fetch origin pull/4/head:ft-add-user-dashboard-page-164976342`
- Now that you have a copy of the branch, switch to it using  `git checkout ft-add-user-dashboard-page-164976342`
- Your directory will now be an exact copy of the PR.
- Cd into user folder by running `cd user/` on terminal
- Then open the `index.html` file with your choice browser

#### What are the relevant pivotal tracker stories?
#164976342
